docs/user-guide/jobs.md

```diff
@@ -206,6 +215,22 @@ If you do not specify `.spec.parallelism`, then it defaults to `.spec.completion
 Depending on the pattern you are using, you will either set `.spec.completions`
 to 1 or to the number of units of work (see [Job Patterns] for an explanation).
 
+### Defaulting Rules for Parallelism and Completions
+
+- `parallelism = nil`, `completions = nil` defaults to `parallelism = 1`, `completions = 1`
```

Comment:
I've fixed the defaults.go.

I'm concerned about the case where a user wants to run a single-completion job.  I don't want them to accidentally scale it up when it is not meant to be scaled up.

I'd rather not make existing Job objects which have neither `completions` or `parallelism` specified suddenly be able to be scaled up above 1.